analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -141,20 +141,22 @@ abstract class PackageManager(
         }
 
         /**
-         * Enrich a package's VCS information with information deduced from the package's VCS URL or an optional
-         * homepage URL.
+         * Enrich a package's VCS information with information deduced from the package's VCS URL or a list of fallback
+         * URLs.
          *
          * @param vcsFromPackage The [VcsInfo] of a [Package].
-         * @param homepageUrl The URL to the homepage of a [Package], if any.
+         * @param fallbackUrls The list of alternative URLs to to use as fallback for determining the [VcsInfo].
          */
-        fun processPackageVcs(vcsFromPackage: VcsInfo, homepageUrl: String = ""): VcsInfo {
+        fun processPackageVcs(vcsFromPackage: VcsInfo, fallbackUrls: List<String> = emptyList()): VcsInfo {
```

Comment:
We should probably reduce the type to `Collection<String>`.

Reply:
The order is important, because the first element that can be used as VCS is taken. I'll add this comment to the docs instead.

Reply:
Done.